### DISCUSSION TITLE
Buttons staying 'highlighted'

### DISCUSSION THREAD

**Discussion by EddRobinson** [tui-cs/Terminal.Gui#2415] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/111066518/225259891-76a06d1a-2501-4727-9bf0-ef043fa96f7a.png)

As you can see in the picture, I have a Wizard view, that contains a Frameview. and then inside there are four buttons. When I click one of the buttons it highlights to show you have clicked it (by default), but in this case, when I click another button, sometimes the button i clicked previously will stay highlighted along with the new button that gets highlighted.

Sometimes all 4 buttons stay highlighted, and sometimes it behaves normally and only the clicked 1 will highlight, other times the one I click doesn't highlight until I click it a second time (but the button still functions as expected on the first click).

But yeah, the buttons still function correctly, despite the 'highlighting', but how would I go fixing / preventing this?

**Comment by BDisp** [comment]
I can't reproduce this. It's seems you have set the `ColorScheme` in a way that the selected button, so the focus attribute, is the one that isn't highlighted.

**Reply by EddRobinson** [reply]
That doesn't seem right to me, as I haven't touched any of the ColorScheme settings. Must be some sort of issue in how my views are set up?

**Reply by BDisp** [reply]
What version do you are using? nuget package, develop or v2_develop branch? It's possible provide some code to test it in your environment?

**Reply by BDisp** [reply]
> Must be some sort of issue in how my views are set up?

I suspect it might be that.

**Comment by EddRobinson** [comment, marked as answer]
Just managed to have a proper look into my issue again after re-structuring a lot of my code (some of my specifications were changed) and found the issue.

It was to do with me basically "refreshing" the list of buttons in a frameview when that button was clicked.

Thankyou for trying to help, I'm slowly getting better
